rust-wasm-id-allocator/wasm-id-allocator/src/lib.rs

```diff
@@ -109,11 +108,11 @@ impl IdCompressor {
         }
     }
 
-    pub fn normalize_local_to_session_space(
-        &mut self,
-        originator_token: f64,
-        op_space_id: f64,
-    ) -> f64 {
+    pub fn normalize_to_session_space(&mut self, op_space_id: f64, originator_token: f64) -> f64 {
+        if originator_token < 0.0 {
```

Comment:
why check for this? I think it's safe to assume (like we do for IDs w.r.t. being integers) that we pass ourselves well-formed input.

Reply:
The reason here was that the next thing we do is a cast to usize and then use that to find the session. So if a negative value does get passed in, it may return a valid session result instead of failing to find as we would want. If we want to assume the contract that a negative value will never be passed, we can remove this.

Reply:
let's assume it's valid, as no external consumer ever affects this value so it's just up to us to not mess it up.

Reply:
Sounds good.